src/agents/coordinator_agent.py

```diff
@@ -80,21 +83,39 @@ class CheckParkingOffers(CyclicBehaviour):
         """Behaviour for checking parking offers"""
 
         def _check_if_request_is_within_region(self, request_x, request_y):
-            return self.agent._x_min <= request_x < self.agent._x_max and self.agent._y_min <= request_y < self.agent._y_max
+            return (
+                self.agent._x_min <= request_x < self.agent._x_max
+                and self.agent._y_min <= request_y < self.agent._y_max
+            )
 
         async def run(self):
-            msg = await self.receive(timeout=MESSAGE_TIMEOUT)
+            # msg = await self.receive(timeout=MESSAGE_TIMEOUT)
+            msg = Message(
+                to="regionalcoordinator1@{DEFAULT_HOST}",
+                body=json.dumps(CheckOffers(0, 1, 0, 1).dict()),
```

Comment:
`json.dumps` results in all key-value pairs being created as strings, e.g. "1":"0" - any better ideas?

Reply:
Do you think it will be problematic? I think we could leave it this way.